packages/modul-components/src/components/photo-editor/photo-editor.html

```diff
@@ -1,6 +1,6 @@
 <m-modal class="m-photo-editor"
          :open="open"
-         :title="i18nTitleModal"
```

Comment:
Il faudrait que tu enlèves cette ligne qui n'est plus utile: i18nTitleModal: string = this.$i18n.translate('m-photo-editor:title');